cluster-scope/overlays/nerc-ocp-test/clusterversion.yaml

```diff
@@ -3,7 +3,7 @@ kind: ClusterVersion
 metadata:
   name: version
 spec:
-  channel: stable-4.11
+  channel: stable-4.12
   desiredUpdate:
-    version: 4.11.46
+    version: 4.12.28
```

Comment:
curious why you went with .28 when the link you provided suggested .27
![image](https://github.com/OCP-on-NERC/nerc-ocp-config/assets/28782106/c287381a-161b-4056-8d1a-32e4cfee3f17)

Reply:
Here is the most current path: [https://access.redhat.com/labs/ocpupgradegraph/update_path?channel=stable-4.10&arch=x86_64&is_show_hot_fix=false&current_ocp_version=4.10.60&target_ocp_version=4.13.8](https://access.redhat.com/labs/ocpupgradegraph/update_path?channel=stable-4.10&arch=x86_64&is_show_hot_fix=false&current_ocp_version=4.10.60&target_ocp_version=4.13.8)

Reply:
When I click on that link I get the path described in the image above.
Here it is again in the CLI section of that link
![image](https://github.com/OCP-on-NERC/nerc-ocp-config/assets/28782106/56247716-b480-4a0d-b76e-24ffba77aaca)
What do you see?

Reply:
@joachimweyl @dystewart if you refresh the page, every other time you do, it will flip-flop versions from 4.12.28 to 4.12.27 to 4.12.28 to 4.12.27. Some kind of weird load balancer problem, hmm.

Reply:
now I see .28, interesting so how do we know which is correct, and does it matter, or are both viable options?